var/spack/repos/builtin/packages/gunrock/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Gunrock(CMakePackage, CudaPackage):
+    """High-Performance Graph Primitives on GPUs"""
+
+    homepage = "https://gunrock.github.io/docs/"
+    url      = "https://github.com/gunrock/gunrock/archive/v1.1.tar.gz"
+    git      = "https://github.com/gunrock/gunrock.git"
+
+    version('develop', git=git, submodules=True)
+    version('1.1', git=git, submodules=True, tag='v1.1', preferred=True)
```

Comment:
No need for `preferred=True` here, Spack will automatically choose the latest numbered release.